packages/accounts-base/change.js

```diff
@@ -0,0 +1,474 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+import { UsersDB } from '../../api/collections/users/server/users';
```

Comment:
This looks like import from a project specific file. I can't find this in the repository.